mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -7927,3 +7950,293 @@ fn test_eth_swap_negotiation_fails_maker_no_fallback() {
     block_on(wait_for_swap_negotiation_failure(&mm_bob, &uuids[0], wait_until));
     block_on(wait_for_swap_negotiation_failure(&mm_alice, &uuids[0], wait_until));
 }
+
+#[derive(Debug, Deserialize)]
+#[serde(deny_unknown_fields, tag = "status", content = "details")]
+pub enum InitTrezorStatus {
+    Ok(InitHwResponse),
+    Error(Json),
+    InProgress(Json),
+    UserActionRequired(Json),
+}
```

Comment:
I tried first but then this InitTrezorStatus struct would refer InitHwResponse struct in mm2_main. Apparently this would invoke some code reorganization

Reply:
> I tried first but then this InitTrezorStatus struct would refer InitHwResponse struct in mm2_main

We usually duplicate structs/enums for tests, so we would duplicate `InitHwResponse` and any other struct/enum that it uses. It's not good practice that's why we started to move the rpc structs/enum to a common crate `mm2src/mm2_rpc/src/data`. If you want to keep this here you can make it non-public since it's not necessary.